packages/scope/index.js

```diff
@@ -32,6 +32,14 @@ export function scope(id) {
 	};
 }
 
+/**
+ * @param {string} path
+ * @returns {string}
+ */
+function createScopeHash(path) {
+	return createHash("sha1").update(path).digest("hex");
```

Comment:
Maybe?
```ts
return `${path.split('/').pop()}-${createHash("sha1").update(path).digest("hex")}`;
```

Reply:
Oh, I'm curious why the `split()` would be better? Using a `/` seems error-prone with windows machines, and I'm not sure why we want the last relative path segment 🤷

Reply:
Sending this to a couple Astro core members to make sure on security. I _think_ this code looks good, just want to be sure!

Reply:
Fair points! I agree we don't need this split thing. And, now that you said, I'm not sure about the safety 😅

Reply:
Done it!